charts/supaglue/templates/_helpers.tpl

```diff
@@ -137,6 +137,29 @@ and we want to make sure that the component is included in the name.
 {{- end -}}
 {{- end -}}
 
+{{- define "supaglue.salesforcePubSub.databaseUrl" -}}
```

Comment:
Can you leave a comment this is experimental, alpha, or for salesforce pubsub only